lib/resources/r4/communication.yaml

```diff
@@ -0,0 +1,193 @@
+---
+name: Communication
+field_name_base_url: http://hl7.org/fhir/r4/communication-definitions.html#Communication
+fields:
+- name: resourceType
+  required: 'Yes'
+  cardinality: 1..1
+  type: string
+  description: The type of the FHIR resource.
+  url: https://hl7.org/fhir/r4/Communication-definitions.html#Communication
+  note: resourceType must be Communication.
+  example: |
+    {
+      "resourceType": "Communication"
+    }
+  action:
+    - create
+
+- name: inResponseTo
```

Comment:
?? shouldn't this be the reply-to extension??

Reply:
Removed inResponseTo: https://github.com/cerner/fhir.cerner.com/pull/433/commits/b51e76543bb04c72ff83076cbc51694bdcb436f1#diff-f08ae1e4c063b9a36a22adaa43fecf21L19

Reply:
@VelocityKnight added ReplyTo in to PR: https://github.com/cerner/fhir.cerner.com/commit/6bfcb726af06ef17098ae7bdac74a5ff2852ac2b#diff-f08ae1e4c063b9a36a22adaa43fecf21R193-R208. Once her codes is merged into mine, it will get added.

Reply:
how about 'A reference to a Group that the recipient should reply to'

Reply:
https://github.com/cerner/fhir.cerner.com/pull/435/commits/3e9a246fcebd45da3dfd9b912b2b32d9eb0e4f73